### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by kerodri**

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ search.appverid: met150
 - [Microsoft Defender for Endpoint Plan 1](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 - [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
 - [Microsoft Defender for Business](/microsoft-365/security/defender-business)
+- Windows 10 (include minimum version)
+- Windows 11 (include minimum version)
+- Does not apply to windows server.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md:29]:
```suggestion
- Windows 10 (include minimum version)
- Windows 11 (include minimum version)
- Does not apply to Windows Server.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kerodri, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — COMMENTED** [review]
Will wait for reviewers comments.

**Comment by denisebmsft** [comment]
Hello @kerodri. I appreciate your feedback and thank you for the pull request. We just updated all the device control documentation for Windows. This article has been replaced by another, so I'm closing this pull request. Thanks again, and please take a look at our updated docs here: https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/device-control-overview?view=o365-worldwide&tabs=Removable

**Pull request closed without merging**